### PR TITLE
Vulnerability fix - crypto to 41.0.0

### PR DESCRIPTION
**Pull request by btam**
bump cryptography to 41.0.0 for vulnerability fix,
bump pyopenssl to 23.2.0 for compatibility with cryptography 41.0.0

### CODE REVIEW

**Review by Michael-Sinz — APPROVED** [review]

**Comment by btam** [comment]
I can't merge this without write access to the repo.

**Pull request merged**
